README.md

```diff
@@ -6,61 +6,199 @@ NOTE: This repository is a WORK IN PROGRESS.
 
 ---
 
-# Datasets Native To This Package
+# Installation
 
-NOTE: Information. Section of the data. How to instantiate. Reason for being. How it was created.
+Install `forecasttools` via:
 
+```
+pip3 install git+https://github.com/CDCgov/forecasttools-py@main
+```
+
+# Vignettes
+
+* Format Arviz Forecast Output For FluSight Submission (In Progress)
+
+# Datasets
+
+By default, `forecasttools` contains several datasets. These datasets aid with experimentation or are directly necessary to some of `forecasttools` utilities.
```

Comment:
What do you mean by "by default" is there a way to disable this?

Reply:
Meant "as `forecasttools` exists currently". Good catch. I will change to: "`forecasttools` contains several datasets."